include/contracts_lite/contract_types.hpp

```diff
@@ -106,9 +120,9 @@ struct ContractViolation {
   }
 
   /** @brief Convenience method to get string version of this object. */
-  std::string string() const {
+  static std::string string(const ContractViolation& cv) {
```

Comment:
Curious if this change (preferring static over the class member function) was driven by performance considerations as well or because this syntax is preferrable?

Reply:
Purely a style preference on my part; I don't think there are any quantifiable reasons for it.